devops/aws/templates/Caddyfile.j2

```diff
@@ -0,0 +1,30 @@
+{{ nip_domain }}/ws-rpc {
+    reverse_proxy localhost:9944
```

Comment:
need to strip  /ws-rpc prefix

Reply:
for http-rpc, colossus and distributor nodes as well

Reply:
Done